CHANGELOG.md

```diff
@@ -2,9 +2,39 @@
 
 ## Table of Contents
 
+- [r0.2.0-rc.2](#r020-rc2)
 - [v0.2.0-rc.1](#v020-rc1)
 - [v0.1.0 - Initial version](#v010---initial-version)
 
+# r0.2.0-rc.2
+
+**This is the second release candidate version for "Identity And Consent Management" release 0.2.0**
+
+## Please note:
+
+* This is a bugfix release.
+
+### Main Changes
+* Fixed the release tag according to Release Management Working Group guidelines.
+* Fixed typo in `access_denied` error for authentication code flow.
+
+### Added
+* N/A
+  
+### Changed
+* N/A
+
+### Fixed
+* Fix returned error in authentication code flow by @garciasolero in https://github.com/camaraproject/IdentityAndConsentManagement/pull/180
```

Comment:
```suggestion
* Fix returned error in authentication code flow by @garciasolero in https://github.com/camaraproject/IdentityAndConsentManagement/pull/180
* Fixed the release tag according to Release Management Working Group guidelines.
```

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/181/commits/ab5ec3c1bd4114c07a4f33e8bbd9c047ac1ebf7e